File: python/day-03.py

arr=[5, 3, 8, 6, 2, 7, 4, 1, 0, 9]
# Lener serch
f=False
target=int(input("Enter the target value: "))
for i in arr:
     if i==target:
          print(f"Target {target} found in the array")
          print(f"Index of the target {target} is :{arr.index(i)}")
          f=True
          break
if f==False:
          print(f"Target {target} not found in the array")

File: cpp/day-03.cpp
#include <iostream>
using namespace std;
int main()
{
     int array[] = {5, 3, 8, 6, 2, 7, 4, 1, 0, 9};
     int n;
     cout << "enter the serching element: ";
     cin >> n;
     int i = 0;
     for (int x : array)
     {
          if (x == n)
          {
               cout << "Element found at index " << (i) << endl;
               break;
          }
          i++;
     }
     cout << "\n";
}